migration-guide/src/explorer-api.md

```diff
@@ -0,0 +1,2904 @@
+              "37btjrVyb4KDsFS7rbQjZQGX6Fz53u76NF2hF9iqRhfbx8ePmjJKCsv5rZV1hhtP6DHdKwLAf5zkVH9FE51xYuCvJzNppSn1tgExVQJpuTwKmyTekT": "19999999999999",
+              "37btjrVyb4KFtfVixJcJdD27YZfXfRM5diZFj4TEazkCAhF8KyztTbe67zQpBc1RTjruHNvefdJ9Jtr7u5rebR71tGrGtKioSSGKy2hMKd6GUVFkEg": "19999999999999",
+              "2cWKMJemoBaiXc4dYCHfDyvy3LcqPebJ8zfRJsZZoHDqik2SzK9Be73YQ5W9u5jEiMPXa": "5428571428571429",
+              "37btjrVyb4KFNpxUYvzqFKsRfRYJKHjEuFfgy6rpoGq29dcMrNhbKTvqNr719U282rp9PcnFonAENkUdv2nE36wZmyNkj8JVQJL1TLu25SKjGFNzVs": "19999999999999",
+              "37btjrVyb4KFYnqsKFfMwj4S3PcCxwxutoiUubHazLfw4wJc5bfrQgpNEpRnGCS2UUfzvMWRRV2vy4wzPKE7tshLS2YEW219R6QfAenXzktMoXCmuU": "19999999999999",
+              "37btjrVyb4KEbo96SLroMfZB11rEvztPswgdqC5ESkHXc4EaFdAhsDQ24vJK23XsjTJzpPS3ZcDWiHiFVtmp5wkJrcnRcMe74s88eTy8YLvLL7EBRj": "19999999999999",
+              "37btjrVyb4KBcyGa22cYKz3Xo9UnB2kzZg4nqVX863JCAvUd54ehdg94DWwnBasCv6sUdbKdh9t4tf48oaXokeoms1HDNuegsmRjVntHBX3Z2hnrV7": "19999999999999",
+              "37btjrVyb4KDHmdZ8ewhythkmGaUzLCwME5pGtWR7nPhE9nCjLMcKstQFyKq1vTSagA2BXtiopfGwtLq2e1jhUVKsw1Z8Me6XTmLm5C9MzRYCZCd3n": "19999999999999",
+              "37btjrVyb4KBGJtifVyePJjSHTno1XNBeTcqmSqEhyUznts9KGQTsvCd9Hq6zM2w29njvmJYCtWmNkUXvayfqyv7epN1awWkKK1WUFQKJsjtevSKz7": "19999999999999",
+              "37btjrVyb4KEwqVLa1yaRPdDUctf525DovPsxoXZfqNp26eXQnmTSFwY3Q3rgPsjRfTKHKtjFpxPy6XYvjzscccsZYfXSaL9MmgrKAaeQgQhCtoXih": "19999999999999",
+              "37btjrVyb4KDxRyqtP9nWyEd7sfzdB8Xgh2egCATJAVtvxM2LhKLp1ALCE714vMCsbQZ5SwvVgiAvmieJTkae865ycwU39JN4pgt27pqEuB8uvi947": "19999999999999",
+              "37btjrVyb4KDA9F68PUv9efaoQHTvacu98Dk6Zx3784ADx4SDnwMfDt3uRfJwqBELeVuis5UEqsf9u4zAD9YC82s6YNmQu43avWDqrQq9Z4hHkEVrL": "19999999999999",
+              "37btjrVyb4KCjschbSccsYGDJo1rVBjdVrpH27iRtc5h1q4XRqpQJTma1NA9t9t8PrTsJjFE7WzNCczJHQR1RGXW1jDNEiEqNa6xctAZ4ZBtXKHVtp": "19999999999999"
+            }
+          }
+        }
+      }
+    }
+    ```
+    </details>
+
+=== "cardano-rosetta"
+
+    !!! warning
+        Not available in Rosetta!
+
+#### [/api/supply/ada](https://input-output-hk.github.io/cardano-rest/explorer-api/#operation/_totalAda)
+
+Get the total ADA supply in the blockchain.
+
+=== "cardano-rest"
+
+    ```console
+    $ https://explorer.cardano-testnet.iohkdev.io/api/supply/ada
+    ```
+
+    <details>
+      <summary>see JSON response</summary>
+    ```json
+    {
+      "Right": 42001932863.01618
+    }
+    ```
+    </details>
+
+=== "cardano-graphql"
+
+    **query.graphql**
+    ```graphql
+    query getAdaSupply {
+      ada {
+        supply {
+          total # corresponds to REST output
+                # total = max - reserves
+          # additional interesting output
+          circulating
+          max
+        }
+      }
+    }
+    ```
+
+    ```console
+    ```
+
+    <details>
+      <summary>see JSON response</summary>
```

Comment:
TODO: I have problems building graphql locally, so cannot get the console cli, nor the json response.